### ISSUE TITLE
Usability question: Finish voting manuall

### ISSUE THREAD

**Issue by bvelte95** [duracellko/planningpoker4azure#103]
Hi there,

I'm really enjoying your work. I'm curious, if there is a possibility for the Scrum Master to close the voting and show the results, even if not everybody has voted yet? That would be neat for my usecase.

**Comment by duracellko** [comment]
Hello,
thank you for your feedback. At the moment it is not possible to finish vote manually. But it is possible to kick out members, who don't vote. And once all members without votes are kicked out, voting is finished.
Scrum Master can see "X" symbol next to each member to remove member from team.

**Comment by duracellko** [comment]
I was avoiding this feature, because making it consistent would require something like 2-phase commit. However, I see it is requested feature, so I am open to proposals.

**Comment by duracellko** [comment]
> [@duracellko](https://github.com/duracellko), Do you mean in the race condition where I vote and the vote is closed nearly at the same time? So on your machine it looks like you voted, but on the Scrum Master's machine the vote was closed first: is that the scenario that you'd think would require a 2-phase commit?
> 
> I suggest there is a simple answer to that: allow those who haven't yet voted after "the vote is closed" to vote anyway. It really can help move the conversation along. Of course, that means that person voting late would be influenced by seeing the votes, but I'd rather risk that than risk not hearing from someone or to have to fiddle with the tool or revote just for one person. Having done hundreds of planning poker sessions, I think this definitely better than any alternative.
> 
> If that's the case, I might just call it something like "reveal estimates" rather than "Close estimation".

Yes, that is the situation, when closing voting and voting at the same time could give different result to scrum master and member.

"Reveal estimation" is interesting feature. I will think about it. Although, technically it just moves the consistency problem from closing voting to starting voting. As the voting is technically never closed and a member votes at the same time as scrum master starts 2nd round, it can happen that for member his vote would count into 1st round, but scrum master would see it in 2nd round.

Anyway, Closing estimation is work in progress.
